code/index.html

```diff
@@ -1,30 +1,227 @@
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
-  </head>
-  <body>
-    <!-- video or image as a header is cool :) -->
-
-    <!-- Signup form -->
-
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
-
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
-      </div>
-
-      <div class="question" id="section2">Section 2</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
-      </div>
+    <title>Dreamzzz</title>
+    <link type="text/css" rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Quicksand:wght@300;400;600&display=swap" rel="stylesheet">
+</head>
+<body>
+
+    <!--VIDEO-->
+
+    <video loop muted autoplay preload="auto">
+        <source src="./videos/NYcleaningconfetti.mp4" video type="video/mp4" />
+    </video>
+
+    <!--HEROIMAGE, HEROTEXT AND OVERLAY TEXT-->
+
+    <div class="hero-image">
+        <div class="hero-text">
+            <h1>You have the dream.<br>We make it happen.</h1>
+        </div>
+    </div>
+
+    <div class="overlay-text">
+        <h1>You have the dream.<br>We make it happen.</h1>
+    </div>
+
+    <!--HEADER-->
+
+    <header>
+        <div class="site-name">The Dream Farm</div>
+        <nav>
+            <a href="#">Home</a>
+            <a href="#">Our dreams</a>
+            <a href="#">Contact</a>
+        </nav>
+    </header>
+
+    <!--INTROTEXT-->
+
+    <section class="intro-text">
+        <h2>Dreams carefully cultivated and exclusively made to order.</h2>
+        <p>Lorem ipsumdolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore
+            magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea
+            commodo consequat. Duis aute irure dolor in.</p>
+    </section>
+
+    <!--SIGN-UP FORM-->
+
+    <section class="container-form-accordion">
+        <form class="form" action="http://httpbin.org/anything" method="POST">
+            <fieldset class="container-form">
+                <legend>
+                    <h2>Harvest Dream</h2>
+                </legend>
+                <div class="form-group">
+                    <label for="firstName">First name*</label>
+                    <input 
+                    id="firstName" 
+                    type="text" 
+                    placeholder="First name" 
+                    name="firstNameValue" 
+                    required
+                    >
+                </div>
+                <div class="form-group">
+                    <label for="lastName">Last name*</label>
+                    <input 
+                    id="lastName" 
+                    type="text" 
+                    placeholder="Last name" 
+                    name="lastNameValue" r
+                    equired
```

Comment:
Nice that you've used the required for the input but this one doesn't work because it's misspelled or divided on two lines? Please check this and be consistent 🙌